linode/config.go

```diff
@@ -95,10 +103,12 @@ func NewConfig(raws ...interface{}) (*Config, []string, error) {
 		errs = packer.MultiErrorAppend(errs, es...)
 	}
 
+	c.Comm.SSHPassword = c.RootPass
```

Comment:
I'm not sure if this was necessary in the end.. Probably wasn't.. I think I tried to run the code without rebuilding it after a different problem.

This would technically be correct if the provisioner had to resort to using the SSHPassword.  I'm not sure if this is the correct time and way to set the c.Comm.SSHPassword though.